src/main/java/Main.java

```diff
@@ -0,0 +1,43 @@
+import entities.SessionStorage;
+import use_cases.session_load.SessionLoadUCI;
+import views.file_session_storage.FileSessionStorage;
+import views.main_menu.MainMenuV;
+import views.session_load.SessionLoadC;
+import views.session_load.SessionLoadP;
+import views.session_load.SessionLoadMenuV;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class Main {
+    public static void main(String[] args) {
+        // Build the main program window
+        JFrame application = new JFrame("Expenditure Tracking");
+        application.setMinimumSize(new Dimension(500, 300));
+        application.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
```

Comment:
I'm not sure if this line is not indented or it's because I'm not reviewing the code in Intellij? Make sure that all lines are indented correctly.

Reply:
I think that might just be a display error? Because it does look indented to me on both IntelliJ and GitHub